db/c.cc

```diff
@@ -4611,18 +4611,18 @@ struct Wrapper : public rocksdb_slicetransform_t {
 };
 
 rocksdb_slicetransform_t* rocksdb_slicetransform_create_fixed_prefix(size_t prefixLen) {
-  Wrapper* wrapper = new Wrapper;
+  SliceTransformWrapper* wrapper = new SliceTransformWrapper;
   wrapper->rep_ = ROCKSDB_NAMESPACE::NewFixedPrefixTransform(prefixLen);
   wrapper->state_ = nullptr;
-  wrapper->destructor_ = &Wrapper::DoNothing;
+  wrapper->destructor_ = &SliceTransformWrapper::DoNothing;
```

Comment:
Not that it is related to this issue, but shouldn't the destructor delete rep?

Reply:
`rocksdb_slicetransform_t` returned by `rocksdb_slicetransform_create_fixed_prefix()` will be managed by a shared_ptr which will eventually call the destructor of `SliceTransformWrapper`, which calls `delete rep_`.

Reply:
This `destructor_` is not used.